src/provider/armTemplates/azuredeploy.json

```diff
@@ -103,6 +103,10 @@
                         {
                             "name": "WEBSITE_NODE_DEFAULT_VERSION",
                             "value": "8.11.1"
```

Comment:
Just occurred to me that we should probably target the latest Node 8.x LTX, v8.16.0, which comes with NPM v.4.1: https://nodejs.org/en/download/releases/

I would assume this is supported?

Reply:
going to put this in a different pr and test :D